tests/integration/tests/live.test.ts

```diff
@@ -0,0 +1,164 @@
+import { describe, expect, test } from "bun:test";
+import { type LiveAction, RecordId, ResponseError, Uuid } from "../../../src";
+import { setupServer } from "../surreal.ts";
+
+const { createSurreal } = await setupServer();
+
+async function withTimeout(p: Promise<void>, ms: number): Promise<void> {
+	const { promise, resolve, reject } = Promise.withResolvers<void>();
+	const timeout = setTimeout(() => reject(new Error("Timeout")), ms);
+
+	await Promise.race([
+		promise,
+		p.then(resolve).finally(() => clearTimeout(timeout)),
+	]);
+}
+
+describe("Live Queries", async () => {
+	const surreal = await createSurreal();
+
+	test("live", async () => {
+		if (surreal.connection?.connection.url?.protocol.startsWith("http")) {
```

Comment:
Avoid conditions inside tests, this can lead to less readable code and worse debugging experience, if the test fail, which path did it took?

You should split to 2 and skip inside if it is not relevant